docs/DeveloperGuide.md

```diff
@@ -181,11 +181,11 @@ Step5. Execution of this method will result in a call to `MainWindow#selectProje
     * Cons:
         * `MainWindow` and `CommandResult` are not closed to modification. A new instruction to change the UI might require the addition of fields to `CommandResult` (boolean fields for instructions and other fields for related data) as well as a new conditional statement in `MainWindow#execute` to handle the new instruction. This makes it relatively difficult to add new instructions.
 
-### Update Commands [Coming soon in v1.3]
```

Comment:
i think  no need the implemented in v1.3  since the other sections also don't have?

Reply:
Sure I'll remove it then. @Eriksen2411's section also has this though.